planetiler-core/src/main/java/com/onthegomap/planetiler/VectorTile.java

```diff
@@ -495,6 +496,23 @@ public byte[] encode() {
     return tile.build().toByteArray();
   }
 
+  /** Returns true if this tile contains only polygon fills (for example, the middle of the ocean). */
+  public boolean containsOnlyPolygonFills() {
+    if (isFill == null) {
+      boolean empty = true;
+      for (var layer : layers.values()) {
+        for (var feature : layer.encodedFeatures) {
+          empty = false;
+          if (!feature.geometry.isFill()) {
+            return false;
```

Comment:
shouldn't this result be cached in `isFilled` as well?

Reply:
good catch